.github/workflows/apply-hotfix.yaml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   resolve-branch:
-    runs-on: ubuntu-latest
+    runs-on: blacksmith-4vcpu-ubuntu-2204
```

Comment:
same here about using the smaller host type (`blacksmith-2vcpu-ubuntu-2204`)... it's just running some shell code.